org.envirocar.app/src/org/envirocar/app/views/dashboard/DashboardFragment.java

```diff
@@ -793,6 +818,61 @@ public void onTextSizeChanged(SizeSyncTextView view, float size) {
         }
     }
 
+    private void spotlightShowCase(View contentView, String head, String content, int nextTarget, int id) {
+        new GuideView.Builder(getContext())
+                .setTitle(head)
+                .setContentText(content)
+                .setGravity(GuideView.Gravity.center)
+                .setTargetView(contentView.findViewById(id))
+                .setContentTextSize(12)
+                .setTitleTextSize(16)
+                .setDismissType(GuideView.DismissType.outside)
+                .setGuideListener(view -> {
+                    switch (nextTarget) {
+                        case 2:
+                            spotlightShowCase(contentView, "Track Details", getString(R.string.dashboard_walkthrough_user_track), 3, R.id.user_statistics);
```

Comment:
Would it be possible to define a String resource with additional German translation for the showcase head?